go.mod

```diff
@@ -22,6 +22,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/apache/pulsar-client-go v0.13.0-candidate-1.0.20240813105849-ab042ae714d1 => ../pulsar-client-go
+
```

Comment:
I'll remove this and change the version once https://github.com/apache/pulsar-client-go/pull/1289 is released